packit_service/service/api/copr_builds.py

```diff
@@ -121,7 +121,7 @@ def get(self, id):
             if project:
                 build_dict["repo_namespace"] = project.namespace
                 build_dict["repo_name"] = project.repo_name
-                build_dict["pr_id"] = build.trigger.get_trigger_object().pr_id
+                build_dict["pr_id"] = build.job_trigger.trigger_id
```

Comment:
This does not look like `pr_id` -- it's a database `id` of the specific trigger object (like `GitBranchModel`, `PullRequestModel`).

If you expect `build` for pull-request -- so the trigger object is `PullRequestModel`, you can use this:

```suggestion
                build_dict["pr_id"] = build.job_trigger.get_trigger_object().pr_id
```

But you need to check, that you get `PullRequestModel` and not e.g. `GitBranchModel`.

---

`job_trigger` is only a mapper between `CoprBuildModel` and specific trigger like `PullRequestModel`, `GitBranchModel`, `ProjectReleaseModel`, ...